packages/graphql/src/translate/translate-create.ts

```diff
@@ -68,57 +67,78 @@ function translateCreate({ context, node }: { context: Context; node: Node }): [
         params: any;
     };
 
-    /* so projection params don't conflict with create params. We only need to call createProjectionAndParams once. */
-    let projAuth = "";
-    const projection = createProjectionAndParams({
-        node,
-        context,
-        fieldsByTypeName,
-        varName: "REPLACE_ME",
-    });
-    if (projection[2]?.authValidateStrs?.length) {
-        projAuth = `CALL apoc.util.validate(NOT(${projection[2].authValidateStrs.join(
-            " AND "
-        )}), "${AUTH_FORBIDDEN_ERROR}", [0])`;
-    }
-
-    const replacedProjectionParams = Object.entries(projection[1]).reduce((res, [key, value]) => {
-        return { ...res, [key.replace("REPLACE_ME", "projection")]: value };
-    }, {});
-
-    if (projection[2]?.connectionFields?.length) {
-        projection[2].connectionFields.forEach((connectionResolveTree) => {
-            const connectionField = node.connectionFields.find(
-                (x) => x.fieldName === connectionResolveTree.name
-            ) as ConnectionField;
-            const connection = createConnectionAndParams({
-                resolveTree: connectionResolveTree,
-                field: connectionField,
-                context,
-                nodeVariable: "REPLACE_ME",
-            });
-            connectionStrs.push(connection[0]);
-            if (!connectionParams) connectionParams = {};
-            connectionParams = { ...connectionParams, ...connection[1] };
+    let replacedProjectionParams: Record<string, unknown> = {};
+    let projectionStr: string | undefined = undefined;
+    let authCalls: string | undefined = undefined;
```

Comment:
I think there is no need to assign these as are `undefined` by default

```suggestion
    let projectionStr: string | undefined;
    let authCalls: string | undefined;
```

Reply:
Good call